Source/Core/Core/AchievementManager.h

```diff
@@ -109,7 +109,7 @@ class AchievementManager
   u32 m_game_id = 0;
   rc_api_fetch_game_data_response_t m_game_data{};
   bool m_is_game_loaded = false;
-  u64 m_last_ping_time = 0;
+  time_t m_last_ping_time = std::time(nullptr);
```

Comment:
Needs an `#include <ctime>`

Reply:
Also I don't think you want to initialize this on class construction, initialize it on game start instead.

Reply:
Alternate idea, what if I initialize it to zero and reset it to zero at game close? That way when the first frame hits, it's already been more than two minutes and immediately generates the first RP.